Sources/Classes/core/MiniApp.swift

```diff
@@ -1,13 +1,33 @@
+import Foundation
+import UIKit
+
 /// Mini App Public API methods
 public class MiniApp: NSObject {
+    public static let version = "4.1.0"
```

Comment:
this variable will be automatically updated by our release script. I added it there because SPM does not provide the possibility to access the package version.

Reply:
Seriously? We don't have a way to retrieve it from Bundle ?

Reply:
Also, can't we use `MiniApp.version` here ?

Reply:
> Seriously? We don't have a way to retrieve it from Bundle ?

Unfortunately not that I know, SPM is till very young, definitely a missing feature